### PR TITLE
Updating SocketsSample

### PR DESCRIPTION
**Pull request by mikaelm12**
Updating SocketsSample with new IHubClients and IHubCallerClients APIs
There was some missing functionality that we can now add easily.

### CODE REVIEW

File: samples/SocketsSample/Hubs/HubTChat.cs [hunk 1]
```diff
@@ -38,14 +48,14 @@ public Task SendToGroup(string groupName, string message)
 
         public async Task LeaveGroup(string groupName)
         {
-            await Groups.RemoveAsync(Context.ConnectionId, groupName);
```
Comment (davidfowl) [inline samples/SocketsSample/Hubs/HubTChat.cs]: Why did you change this order?

Comment (mikaelm12) [reply samples/SocketsSample/Hubs/HubTChat.cs]: So that you see the "left group" message when you leave.
